tests/netflow_shorten_ipv6/test.yaml

```diff
@@ -0,0 +1,48 @@
+
+args:
+- -k none --set logging.ipv6-addr-shorten=yes -vvv
+
+checks:
+- filter:
+    count: 1
+    match:
+      app_proto: http
+      src_ip: fe80::20c:29ff:fef3:cf38
```

Comment:
Could we focus on the important fields (we do not care about `tcp.ack` here, do we ?)